lib/coinbase/wallet.rb

```diff
@@ -17,6 +17,17 @@ class Wallet
     # The maximum number of addresses in a Wallet.
     MAX_ADDRESSES = 20
 
+    # A representation of ServerSigner status in a Wallet.
+    module ServerSignerStatus
+      # The Wallet is awaiting seed creation by the ServerSigner. At this point,
+      # the Wallet cannot create addresses or sign transactions.
+      PENDING_SEED_CREATION = :pending_seed_creation
```

Comment:
Just `PENDING`?

And do we need `FAILED`?

Reply:
There is no "FAILED" state yet on the backend. A failure for the SDK will be a timeout

Reply:
PENDING conflicts with the transferStatus constants. Hence renamed this to `pending_seed_creation` on backend

Reply:
They are scoped in different modules so there should be no conflict

Reply:
renamed at the SDK level